l1-contracts/src/core/RollupCore.sol

```diff
@@ -59,7 +59,7 @@ contract RollupCore is
   uint256 public immutable L1_BLOCK_AT_GENESIS;
 
   // To push checkBlob into its own slot so we don't have the trouble of being in the middle of a slot
-  uint256 private gap = 0;
+  uint256 private gap = 0; // solhint-disable-line private-vars-leading-underscore
```

Comment:
This should probably be fixed in your settings instead 🤔 we are not applying this rule, if you look in your vs-code you are probably using a different set of rules than what we have in our solhint file.